pkg/tests/ossm/operator/clusterwide_mode_test.go

```diff
@@ -375,26 +375,20 @@ spec:
 			oc.WaitSMCPReady(t, meshNamespace, smcpName)
 
 			t.LogStep("Check if mTLS is enabled in foo")
-			retry.UntilSuccess(t, func(t test.TestHelper) {
-				oc.Exec(t,
-					pod.MatchingSelector("app=sleep", "foo"),
-					"sleep",
-					"curl http://httpbin.foo:8000/headers -s",
-					assert.OutputContains("X-Forwarded-Client-Cert",
-						"mTLS is enabled in namespace foo (X-Forwarded-Client-Cert header is present)",
-						"mTLS is not enabled in namespace foo (X-Forwarded-Client-Cert header is not present)"))
-			})
```

Comment:
This is great but the retry.UntilSuccess func is there to avoid flaky results. This will retry whatever you put inside the func, so you have two options: 1) Put again the retry.UntilSuccess or 2) See if you can add the retry inside the app.ExecInSleepPod to do it everytime you use that func

Reply:
Ohhhh, sorry I did not notice. Great then :)